shell/app/modules/project/pages/test-manage/case/project-tree-modal/index.tsx

```diff
@@ -31,15 +31,14 @@ const ProjectTreeModal = () => {
   const [confirmLoading] = useLoading(testSetStore, ['submitTreeModal']);
   const { submitTreeModal } = testSetStore.effects;
   const { closeTreeModal, updateTreeModalExtra } = testSetStore.reducers;
-
   const { action, type } = treeModalInfo;
   const visible = !!type;
   return (
     <Modal
       visible={visible}
       title={titleMap[action]}
       onCancel={closeTreeModal}
-      onOk={submitTreeModal}
+      onOk={() => submitTreeModal()}
```

Comment:
What's the difference? `submitTreeModal` does not accept any arguments. It's better to write like this but not solve the bug?

Reply:
Prevent pass redundant arguments to effect, prevent immer get it and throw error.